src/index.html

```diff
@@ -5,8 +5,351 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a class="header__nav-logo-link" href="#">
+          <img
+            src="images/moyo_1.png"
+            alt="moyo_img"
+            class="header__nav-logo"
+          >
+        </a>
+        <nav>
+          <ul class="header__nav-list">
+            <li class="header__nav-item header__nav-item--is-active">
+              <a class="is-active" href="#Apple">Apple</a>
+            </li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Samsung">Samsung</a></li>
```

Comment:
Fix it code style everywhere
```suggestion
            <li class="header__nav-item">
              <a class="header__nav-link" href="#Samsung">Samsung</a>
            </li>
```